internal/ark/config.go

```diff
@@ -156,19 +156,14 @@ func (req ConfigRequest) Get() (values ValueOverrides, err error) {
 		return values, err
 	}
 
-	rbacEnabled := true
-	if req.Cluster.Provider == providers.Azure {
-		rbacEnabled = false
-	}
-
 	return ValueOverrides{
 		Configuration: configuration{
 			PersistentVolumeProvider: pvp,
 			BackupStorageProvider:    bsp,
 			RestoreOnlyMode:          req.RestoreMode,
 		},
 		RBAC: rbac{
-			Create: rbacEnabled,
+			Create: true,
```

Comment:
This should come from the cluster's RbacEnabled property.

Reply:
👍